01_getting_started/hello_world.py

```diff
@@ -49,13 +46,13 @@ def f(i):
 #
 # Now let's see three different ways we can call that function:
 #
-# 1. As a regular `local` call on your computer, with `f.local`
+# 1. As a regular `local` call on your computer, with `f.local()`
 #
-# 2. As a `remote` call that runs in the cloud, with `f.remote`
+# 2. As a `remote` call that runs in the cloud, with `f.remote()`
 #
-# 3. By `map`ping many copies of `f` in the cloud over many inputs, with `f.map`
```

Comment:
The use of backticks around `map` here is intentional -- it is symmetric with `local` and `remote` in the other two bullet points.

The point is to help the reader connect the specific Modal objects (`.remote`, `.local`, `.map`) with concepts they already understand (remote and local execution, mapping a function over many inputs).

Reply:
Not everyone knows what a map is